IO-phone/src/config.ts

```diff
@@ -0,0 +1,4 @@
+// TODO: this config will not work in deployment since the the app is ran on different device than the server!
+export const API_PORT = "8000"
+export const API_HOST = "localhost";
```

Comment:
To będzie to samo IPv4, którego wyświetlanie zakodziłeś, co nie?
Dodałbyś proszę ticket, żeby to pobierać i tej wartości używać?